pytestqt/plugin.py

```diff
@@ -210,6 +216,12 @@ def pytest_configure(config):
 
     qt_api.set_qt_api(config.getini("qt_api"))
 
+    if config.getini("qt_wait_signal_raising"):
```

Comment:
Hmm, I think it's a string (because there's `_parse_ini_boolean` in `qtbot.py`), but I wonder why we don't pass `type="bool"` in the `parser.addini` calls in `pytest_addoption` to let pytest do the ini parsing?

Reply:
No idea... 🤔

Reply:
But if you see the warning if `qt_wait_signal_raising` is set to False or True, I'm happy!